src/main/java/gregtechfoodoption/recipe/chain/BritishChain.java

```diff
@@ -16,6 +16,13 @@
 
 public class BritishChain {
     public static void init() {
+        RecipeMaps.CHEMICAL_RECIPES.recipeBuilder().EUt(30).duration(50)
```

Comment:
While the formula Na2CO3 + CO2 + H2O -> 2NaHCO3 is stoichiometrically correct, this doesn't follow GTCEu's conventions. Particularly, the fluids should all be multiples of 1000, and the dusts should be used in quantities that are a multiple of the number of atoms that make up each molecule, just like most GTCEu recipes. At the very least, this recipe does seem to exist in real life.